docs/release_notes/release_notes.md

```diff
@@ -9,41 +9,60 @@ has_children: true
 # Release Notes
 
 
-## Version 4.7.0 (March 2024)
+## Version 4.8.0 (May 2024)
 
-* ### RSA Radius Authentication
-    * Added new dashboard named **RSA Radius Authentication**.
-    * Added new alert named **RSA Radius Authentication - Excessive Failed Logins for a User**.
-
-* ### Office 365 Alerts/Dashboard
+* ### Cisco Meraki
     * Added new alerts:
-        * O365 - OneDrive or SharePoint File Sharing with External User
-        * O365 - OneDrive or SharePoint Link Accessed By External User
-    * Added panels related to mentioned alerts on **Office 365** dashboard.
+        * Cisco Meraki - Organizational Security Events
+        * Cisco Meraki - Config Changes
+    * Added new dashboard named **Cisco Meraki**
 
-* ### Google Workspace Alerts/Dashboard
-    * Added new alerts:
-        * Google Workspace - Google Drive objects accessed by External User
-    * Added panels related to mentioned alerts on **Google Workspace** dashboard.
+* ### Databases
+    * #### MSSQL
+        * Added new alert named **MSSQL - User Changes**
```

Comment:
Is all the alerts and dashboards added to doc's page??

Reply:
yes, it covers all alert and dashboard and it's added as part of other doc specific PRs.